backend/routes/api.php

```diff
@@ -202,6 +202,7 @@ function () {
 // Interview routes
 Route::prefix('interview')->group(function () {
     Route::get('all', [InterviewController::class, 'getInterviews']);
+    Route::get('/admin/all', [InterviewController::class, 'getInterviewsAdmin'])->middleware("isloggedin", "isadmin");
```

Comment:
please admin route can't be created this way. Create a sub group for interview under the admin grouped routes